govtool/frontend/src/i18n/locales/en.ts

```diff
@@ -267,6 +267,8 @@ export const en = {
       goToDRepDirectory: "Go to DRep Directory",
       meAsDRep: "This DRep ID is connected to your wallet",
       myDelegation: "You have delegated your voting power to:",
+      myDelegationToYourself:
+        "You have delegated your voting power to yourself:",
```

Comment:
Just to be sure: a colon not a period at the end of sentence?

Reply:
It's on a chip, so maybe nothing would make most sense. But on the other hand, that's the way it used to be before I deleted it prematurely a few PRs back